### PR TITLE
Add Docusaurus Docs

### PR DESCRIPTION
**Pull request by strseb**
## Description

mozillavpn.dev

### CODE REVIEW

File: tools/mozillavpn.dev/src/plugins/resources.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+if(!existsSync(RES_OUT_FOLDER)){
+  mkdirSync(RES_OUT_FOLDER)
+}
+
+export default function(context, options) {
+  return {
+    name: 'copy-res-plugin',
+    loadContent() {
+      /**
+       * Copy the Constents of the clients ui/res to
```
Comment (lesleyjanenorton) [inline tools/mozillavpn.dev/src/plugins/resources.ts:29]: content? constants, maybe?

File: tools/mozillavpn.dev/src/sidebars/default.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+ Create as many sidebars as you want.
+ */
+const sidebars: SidebarsConfig = {
+  // By default, Docusaurus generates a sidebar from the docs folder structure
+  tutorialSidebar: [{type: 'autogenerated', dirName: '.'}],
+
+  // But you can create a sidebar manually
+  /*
+  tutorialSidebar: [
```
Comment (lesleyjanenorton) [inline tools/mozillavpn.dev/src/sidebars/default.ts:19]: Any reason to keep this?

**Review by lesleyjanenorton — COMMENTED** [review]
🤩

File: docs/rfcs/0005-suggested-excluded-apps-tech-spec.md [hunk 3]
```diff
@@ -45,12 +45,12 @@ This work is broken into four sections, with the bulk of code coming in the midd
     -   Additions to the list should be given the same amount of thought that adding new dependencies to our repo are given. We should err on the side of not including any.
         -   This statement (or something similar) should be added to the top of the file. This will be in some JSON value that serves as a comment.
         -   The precise methods to determine whether an app belongs on the list is described in the [epic](https://mozilla-hub.atlassian.net/browse/VPN-4412).
-    -   From Santiago (this is a reminder to review these later): <https://github.com/citizenlab/test-lists/tree/master/lists>, <https://ooni.org/>, for Android <https://docs.google.com/spreadsheets/d/1j2REPQkcdPg5bNW5B3T-L8dnlNPPwAms1WlBiYvVwoc/edit#gid=0>
+    -   From Santiago (this is a reminder to review these later): https://github.com/citizenlab/test-lists/tree/master/lists, https://ooni.org/ for Android https://docs.google.com/spreadsheets/d/1j2REPQkcdPg5bNW5B3T-L8dnlNPPwAms1WlBiYvVwoc/edit#gid=0
```
[the remark this answers is not in the document]
Comment (strseb) [reply docs/rfcs/0005-suggested-excluded-apps-tech-spec.md:48]: They seem to render the same [main](https://github.com/mozilla-mobile/mozilla-vpn-client/blob/main/docs/rfcs/0005-suggested-excluded-apps-tech-spec.md#prerequisites-15-26-points-total) || [docusaurus branch](https://github.com/mozilla-mobile/mozilla-vpn-client/blob/docusaurus/docs/rfcs/0005-suggested-excluded-apps-tech-spec.md#prerequisites-15-26-points-total)
This was not valid markdown, given `< >` is used for inline html. 

Future docs will not be reformatted, but during docusaurus build we will check for markdown validity and that there are no dead links.  (build will fail if not correct)

**Review by brizental — APPROVED** [review]

**Pull request merged**